launchpod/angular-launchpod/backend/src/test/java/com/google/launchpod/FormHandlerServletTest.java

```diff
@@ -167,21 +195,36 @@ public void doPost_ReturnsCorrectUrl() throws IOException {
 
     assertEquals(1, ds.prepare(new Query(USER_FEED)).countEntities(withLimit(10)));
 
-    Query query = new Query(USER_FEED);
-    PreparedQuery preparedQuery = ds.prepare(query);
-    Entity desiredEntity = preparedQuery.asSingleEntity();
-
-    String expectedXmlString = RSS.toXmlString(TEST_RSS_FEED);
-    assertEquals(expectedXmlString, desiredEntity.getProperty(XML_STRING).toString());
-
-    String testXmlString = RSS.toXmlString(TEST_RSS_FEED);
-    assertEquals(testXmlString, desiredEntity.getProperty(XML_STRING).toString());
-
-    String id = KeyFactory.keyToString(desiredEntity.getKey());
-    String rssLink = BASE_URL + id;
-
-    verify(response, times(1)).setContentType("text/html");
-    assertEquals(0, rssLink.compareTo(stringWriter.toString()));
+    Query query =
+        new Query(USER_FEED).addSort(TIMESTAMP, SortDirection.DESCENDING);
```

Comment:
you could also filter on user in this query too right?

Reply:
updated with merge from previous branch.